.taskmaster/error-handler.py

```
@@ -0,0 +1,350 @@
+                            with open(pid_file, 'r') as f:
+                                pid = int(f.read().strip())
+                            
+                            if not psutil.pid_exists(pid):
+                                self.log_error(
+                                    ErrorType.AGENT_CRASH,
+                                    ErrorSeverity.MEDIUM,
+                                    f"Agent {agent} process crashed (PID: {pid})",
+                                    agent_id=agent,
+                                    details={"crashed_pid": pid}
+                                )
+                        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)